### PR TITLE
feat: add C implementation for `stats/base/dists/arcsine/kurtosis`

### LINKED ISSUE #3385: [RFC]: Add C implementation for `@stdlib/stats/base/dists/arcsine/kurtosis`
### Description

This RFC proposes adding a C implementation, including a Node.js native addon, C benchmarks, and C examples, for the [`@stdlib/stats/base/dists/arcsine/kurtosis`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dists/arcsine/kurtosis) package.

When adding support, the following tasks should be completed:

1. A C source implementation should be added to a `src` folder within the package's directory.
2. A native add-on interface should be added in order to call the C implementation from JavaScript.
3. A JavaScript file should be added to the `lib` folder to allow benchmarking and unit testing the native implementation from JavaScript.
4. A new test file should be created which includes unit tests testing against expected behavior.
5. C benchmarks should be added which measure performance for the C source implementation.
6. A file containing C examples should be added to demonstrate example usage.
7. JavaScript benchmarks should be added which measure performance for the native add-on.
8. Build and configuration files should be added to allow compiling and running the C source implementation using project `make` commands (as documented below).
9. The README should be updated to include documentation for the added C API.

To provide a concrete example of what a PR adding the desired method should contain, see https://github.com/stdlib-js/stdlib/pull/3354, which is a PR adding a C implementation for the CDF of an arcsine distribution. This should provide an idea of what is expected.

#### Prerequisites

1. Review JavaScript implementation in `lib/main.js` and its required modules to identify all `stdlib` functions used.
2. Verify that C implementations exist for each required function.

### Related Issues

#3355

### Questions

No.

### Other

Once the implementation is ready including C examples and benchmarks, we should be able to run the following `make` commands.

#### Build native add-on

```bash
NODE_ADDONS_PATTERN="@stdlib/stats/base/dists/arcsine/kurtosis" make install-node-addons
```

#### Run C Examples

```bash
make examples-c-files FILES="$(pwd)/lib/node_modules/@stdlib/stats/base/dists/arcsine/kurtosis/examples/c/example.c"
```

#### Run C benchmarks

```bash
make benchmark-c-files FILES="$(pwd)/lib/node_modules/@stdlib/stats/base/dists/arcsine/kurtosis/benchmark/c/benchmark.c"
```

#### Run JavaScript benchmarks

```bash
make benchmark-javascript-files FILES="$(pwd)/lib/node_modules/@stdlib/stats/base/dists/arcsine/kurtosis/benchmark/benchmark.native.js"
```

#### Run All Tests

```bash
make TESTS_FILTER=".*/@stdlib/stats/base/dists/arcsine/kurtosis/.*" test
```

### Notes

- If you are interested in contributing a PR which addresses this RFC and are a **first-time contributor** or not yet familiar with our project conventions, please minimize the submission of LLM-generated code. Please consult our [contributing guidelines](https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md) and the associated [development guide](https://github.com/stdlib-js/stdlib/blob/develop/docs/development.md). Failure to respect project conventions will result in your PR being rejected without review. Thank you for understanding!

### Checklist

- [x] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [x] Searched for existing issues and pull requests.
- [x] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by aayush0325**
Resolves #3385

## Description

> What is the purpose of this pull request?

This pull request:

-   adds C implementation for `@stdlib/stats/base/dists/arcsine/kurtosis` along with relevant tests, benchmarks, docs and examples.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #3385

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by aayush0325** [comment]
greetings! please give this a review @Planeshifter

**Comment by aayush0325** [comment]
There was a missing `benchmark.native.js` file here, my apologies, pushed it now.

File: lib/node_modules/@stdlib/stats/base/dists/arcsine/kurtosis/README.md [hunk 1]
```diff
@@ -131,6 +131,101 @@ for ( i = 0; i < 10; i++ ) {
...
+#include "stdlib/stats/base/dists/arcsine/kurtosis.h"
+```
+
+#### stdlib_base_dists_arcsine_kurtosis( a, b )
+
+Returns the excess kurtosis of an arcsine distribution.
+
+```c
+double out = stdlib_base_dists_arcsine_kurtosis( 0.0, 2.0 );
+// returns 1.5
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/stats/base/dists/arcsine/kurtosis/README.md]: Return value was incorrect:

```suggestion
// returns -1.5
```

File: lib/node_modules/@stdlib/stats/base/dists/arcsine/kurtosis/benchmark/benchmark.native.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+var kurtosis = tryRequire( resolve( __dirname, './../lib/native.js' ) );
+var opts = {
+	'skip': ( kurtosis instanceof Error )
+};
+
+
+// MAIN //
+
+bench( pkg, opts, function benchmark( b ) {
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/stats/base/dists/arcsine/kurtosis/benchmark/benchmark.native.js]:
```suggestion
bench( pkg+'::native', opts, function benchmark( b ) {
```

**Review by Planeshifter — APPROVED** [review]
Thanks for your contribution, @aayush0325!

**Pull request merged**